qutebrowser/test/mainwindow/statusbar/test_textbase.py

```diff
@@ -0,0 +1,46 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2014-2015 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+
+"""Test TextBase widget."""
+
+from qutebrowser.mainwindow.statusbar.textbase import TextBase
+
+
+def test_elided_text(qtbot):
+    """
+    Ensure that a widget that can't hold the entire label text will display
+    and elided version of the string instead.
+
+    Args:
+        qtbot: pytestqt.plugin.QtBot fixture
+
+    Note:
+
+    It is difficult to check what is actually being drawn in a
+    portable way, so at least we ensure our customized methods are being
+    called and the elided string contains the horizontal ellipsis character.
```

Comment:
You say "we ensure our customized methods are being called" here - though I don't understand what that means exactly - the test just checks the string contains the ellipsis, right?